### ISSUE TITLE
Issue: Incorrect Quote Conversion in XML to JSON Conversion

### ISSUE THREAD

**Issue by Schokojim** [nirgit/simple-xml-to-json#50]
#### Description
In my XML, I have the following value:
```xml
<cbc:StreetName>D'Amore Divide</cbc:StreetName>
```

When I call the `xml2Json.convertXML(xml)` function, I get the following result:
```json
{
  "content": "D\"Amore Divide"
}
```

#### Problem
The single quote (`'`) is incorrectly converted to a double quote (`"`).

#### Steps to Reproduce
1. Create an XML with the value `<cbc:StreetName>D'Amore Divide</cbc:StreetName>`.
2. Use the `xml2Json.convertXML(xml)` function to convert the XML to JSON.
3. Observe the conversion result.

#### Expected Behavior
The single quote should remain as a single quote in the JSON output:
```json
{
  "content": "D'Amore Divide"
}
```

#### Actual Behavior
The single quote is converted to a double quote:
```json
{
  "content": "D\"Amore Divide"
}
```

#### Environment
- XML to JSON Conversion Tool: `simple-xml-to-json`
- Version: 1.2.2
- Operating System: Win11 64bit
- node version: 20.14.0

**Comment by Aymkdn** [comment]
Same issue here…

**Comment by nirgit** [comment]
thanks for reporting the bug @Schokojim and @Aymkdn 

fix for the bug will likely be merged soon https://github.com/nirgit/simple-xml-to-json/pull/53

you're welcome to test it and leave your feedback, thanks

**Comment by nirgit** [comment]
issue should be resolved in latest v1.2.3
